CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+CMAKE_MINIMUM_REQUIRED (VERSION 2.6)
+
+#add_definitions(-DOEM_TYPE=${OEM})
+
+#Required Packages
+PROJECT(leelaz)
+
+SET (BOOST_MIN_VERSION "1.58.0")
+FIND_PACKAGE(Boost ${BOOST_MIN_VERSION} REQUIRED)
+FIND_PACKAGE( BLAS REQUIRED )
+FIND_PACKAGE( ZLIB REQUIRED )
+FIND_PACKAGE( OpenCL REQUIRED )
+
+SET(IncludePath "CMAKE_CURRENT_SOURCE_DIR}/src")
+SET(SrcPath "${CMAKE_CURRENT_SOURCE_DIR}/src")
+
+INCLUDE_DIRECTORIES(${IncludePath})
+INCLUDE_DIRECTORIES(/usr/include/openblas)
+
+SET(SOURCES
+    "${SrcPath}/FastBoard.cpp"
```

Comment:
Can this be automated via some kind of glob? Everything in src/*.cpp should be same to compile.